src/pages/admin/addHotel/addHotel.js

```diff
@@ -0,0 +1,13 @@
+import AddHotelForm from '../../../components/AddHotel/AddHotelForm';
+
+const AddHotel = () => (
+  <section>
+    <div className="d-flex justify-content-center">
+      <h1>Add Hotel</h1>
+    </div>
```

Comment:
```suggestion
    <div className="text-center my-5">
      <h1>Add Hotel</h1>
    </div>
```

Reply:
- [Optional] I think the page would look better if I added some margin (top, bottom). 
- If you agree with this suggestion, you can just commit this change automatically.